tmcprototype/SubarrayNode/SubarrayNode/SubarrayNode.py

```diff
@@ -948,6 +948,16 @@ def setPointingState(self, evt):
             "Subarray Node.",
     )
 
+    CspSubarrayNodeFQDN = device_property(
+        dtype='str', default_value="mid_csp/elt/subarray_01"
+    )
+
+    SdpSubarrayNodeFQDN = device_property(
+        dtype='str', default_value="mid_sdp/elt/subarray_1"
```

Comment:
Use variables from CONST.py file instead of hardcoded string in code.

Reply:
It has been changed as per the suggestions.
commit ID: [47c87a46eff5142110ead02b9377b5c29e5c655d](https://github.com/ska-telescope/tmc-prototype/pull/40/commits/47c87a46eff5142110ead02b9377b5c29e5c655d)